packages/core/testutils/src/JestEventSourceEventStream.ts

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright 2018-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  BaseEventStream,
+  Event,
+  StitchClientError,
+  StitchClientErrorCode, 
+  StitchEvent
+} from "mongodb-stitch-core-sdk";
+
+/** @hidden */
+export default class JestEventSourceEventStream extends BaseEventStream<JestEventSourceEventStream> {
```

Comment:
[broader discussion] So this works and I know it's what I told you to do, but now this means that the Transports we wrote are not covered at all by our tests unless we do E2E browser, Node.js, and React Native testing.

We aren't going to make changes to the other transports again for a while probably, but it might be a good idea to file a backlog ticket to do some very rudimentary E2E smoke testing with an actual node server, actual react native app, and actual browser (via something like selenium) to make sure the transports properly interact with the network.

Reply:
thoughts @jsflax ?

Reply:
Consider how core of an element the transports are, I think we should add some basic tests to make sure everything still works, before merging this. More can be added later.

Reply:
The point is that these transports can't be tested in Jest, so we'd have to create an entire E2E testing strategy, which I feel is out of scope for this ticket, which was meant to fix a bug.

Tyler did do some manual E2E in the browser to make sure the transports still work, and I think it would be good to merge if he did the same for Server and RN

Reply:
Ah, fair point. That works for me.